### PR TITLE
Feature/SM-1 Implemented the login story

### PR DESCRIPTION
**Pull request by senciuckristine**

### CODE REVIEW

**Comment by dragosremetea** [comment]
Please resolve conflicts!

File: src/loose/oose/fis/lab/student/manager/controllers/Controller.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package loose.oose.fis.lab.student.manager.controllers;
+
+public class Controller {
+}
```
Comment (dragosremetea) [inline src/loose/oose/fis/lab/student/manager/controllers/Controller.java:4]: Remove unuesd file

File: src/loose/oose/fis/lab/student/manager/fxml/sample.fxml [hunk 2]
```diff
@@ -3,6 +3,6 @@
 
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.control.Label?>
-<GridPane fx:controller="sample.Controller"
+<GridPane fx:controller="loose.oose.fis.lab.student.manager.controllers.Controller"
           xmlns:fx="http://javafx.com/fxml" alignment="center" hgap="10" vgap="10">
 </GridPane>
```
Comment (dragosremetea) [inline src/loose/oose/fis/lab/student/manager/fxml/sample.fxml:8]: Delete unused file

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            return;
+        }
+
+        if (username.equals("student") && password.equals("student")) {
+            loginMessage.setText("Logged in as student!");
+            return;
+        }
+
+        if (username.equals("teacher") && password.equals("teacher")) {
+            loginMessage.setText("Logged in as teacher!");
```
Comment (dragosremetea) [inline src/loose/oose/fis/lab/student/manager/controllers/LoginController.java:38]: If login as a teacher succeeds, the user should be taken to the view-students scene

File: src/loose/oose/fis/lab/student/manager/fxml/sample.fxml (hunk 2 above)
Comment (senciuckristine) [reply src/loose/oose/fis/lab/student/manager/fxml/sample.fxml:8]: Done.

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java (hunk 3 above)
Comment (senciuckristine) [reply src/loose/oose/fis/lab/student/manager/controllers/LoginController.java:38]: Done.

**Pull request merged**